### DISCUSSION TITLE
replace from manim import *

### DISCUSSION THREAD

**Discussion by nicataliyev** [ManimCommunity/manim#2639] [Architecture]
Hi dear manim team,

Are there any plans to change the way manim vars and modules are imported using `import *`. Since `import *` is considered to be bad practice from python software engineering perspectice and sometimes causes confusions when using custom modules as those module names may interfere with sub-modules of manim.

**Comment by behackl** [comment]
Hello! There are no short-term plans to change how we use the library in our documentation I'd say -- but you don't *have* to use `*`-imports anyways, you can always use scripts like this:

```py
import manim as mn

class HelloWorld(mn.Scene):
    def construct(self):
        t = mn.Text("Hello World!")
        self.play(mn.Write(t))
        self.play(mn.FadeOut(t))
```
